presto-main/src/test/java/com/facebook/presto/sql/query/TestAggregationOverJoin.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.query;
+
+import org.testng.annotations.Test;
+
+public class TestAggregationOverJoin
+{
+    @Test
+    public void test()
```

Comment:
Instead of creating new test class per single class, maybe there is (or could be created) `TestJoinQueries`, so you could create there `testAggregationOverJoin`

Reply:
I couldn't find any good place to put this. TestJoinQueries is overly generic. I settled on adding a new specialized test for now. Eventually we can merge it with others if a theme arises.